pkg/agent/agent.go

```diff
@@ -901,3 +916,21 @@ func (i *Initializer) allocateGatewayAddresses(localSubnets []*net.IPNet, gatewa
 
 	return nil
 }
+
+func (i *Initializer) patchNodeAnnotations(nodeName, key string, value interface{}) error {
+	patch, _ := json.Marshal(map[string]interface{}{
+		"metadata": map[string]interface{}{
+			"annotations": map[string]interface{}{
+				key: value,
+			},
+		},
+	})
+	if err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
+		_, err := i.client.CoreV1().Nodes().Patch(context.TODO(), nodeName, apitypes.MergePatchType, patch, metav1.PatchOptions{})
+		return err
+	}); err != nil {
+		klog.ErrorS(err, "Failed to patch Node annotation:", "key", key, "value", value)
```

Comment:
":" is not needed, otherwise it would look weird. Maybe you should check the log file to see how the structured logging formats the message.

Reply:
Sure, I will remove it.